Jenkinsfile

```diff
@@ -13,189 +13,85 @@
     CHART_DESCRIPTOR = "${CHART_NAME}/Chart.yaml"
     CHART_SERVICE = 'http://chartmuseum:8080'
-    TEST_NAMESPACE = "nuxeo-helm-chart-${BRANCH_NAME}-${BUILD_NUMBER}".toLowerCase()
     TEST_RELEASE = 'test-release'
     TEST_K8S_RESSOURCE = "${TEST_RELEASE}-${CHART_NAME}"
-    TEST_SERVICE_DOMAIN = "${TEST_K8S_RESSOURCE}.${TEST_NAMESPACE}.svc.cluster.local"
-    TEST_ROLLOUT_STATUS_TIMEOUT = '5m'
-    BUILD_VERSION = getBuildVersion("${CHART_DESCRIPTOR}")
-    CHART_ARCHIVE = "${CHART_NAME}-${BUILD_VERSION}.tgz"
+    VERSION = nxUtils.getVersion(baseVersion: getChartVersion("${CHART_NAME}"), tagPrefix: '')
+    CHART_ARCHIVE = "${CHART_NAME}-${VERSION}.tgz"
   }
   stages {
     stage('Helm package') {
       steps {
-        setGitHubBuildStatus('package', 'Package and upload Helm chart', 'PENDING')
         container('base') {
-          script {
-            currentBuild.description = "${BUILD_VERSION}"
-            updateVersion("${BUILD_VERSION}")
+          nxWithGitHubStatus(context: 'package', message: 'Package and upload Helm chart') {
+            script {
+              currentBuild.description = "${VERSION}"
+              echo "Update chart to version ${VERSION}"
+              sh "yq write -i ${CHART_DESCRIPTOR} version ${VERSION}"
 
-            echo "Package chart version: ${BUILD_VERSION}"
-            sh "helm3 package ${CHART_NAME}"
+              echo "Package chart version: ${VERSION}"
+              sh "helm3 package ${CHART_NAME}"
 
-            echo 'Test chart'
-            // install the chart into a test namespace that will be cleaned up afterwards
-            sh "kubectl create namespace ${TEST_NAMESPACE}"
-            sh "kubectl --namespace=platform get secret kubernetes-docker-cfg -ojsonpath='{.data.\\.dockerconfigjson}' | base64 --decode > /tmp/config.json"
-            sh """
-              kubectl create secret generic kubernetes-docker-cfg \
-                --namespace=${TEST_NAMESPACE} \
-                --from-file=.dockerconfigjson=/tmp/config.json \
-                --type=kubernetes.io/dockerconfigjson --dry-run -o yaml | kubectl apply -f -
-            """
-            sh """
-              helm3 install ${TEST_RELEASE} ${CHART_NAME} \
-                --namespace=${TEST_NAMESPACE} \
-                --values=ci/values.yaml
-            """
-            try {
-              // check deployment status, exit if not OK
-              sh """
-                kubectl rollout status deployment ${TEST_K8S_RESSOURCE} \
-                  --namespace=${TEST_NAMESPACE} \
-                  --timeout=${TEST_ROLLOUT_STATUS_TIMEOUT}
-              """
-              // check running status
-              sh "ci/running-status.sh http://${TEST_SERVICE_DOMAIN}/nuxeo"
-            } catch (e) {
-              sh """
-                kubectl --namespace=${TEST_NAMESPACE} get all,configmaps,endpoints,ingresses
-                kubectl --namespace=${TEST_NAMESPACE} describe pod --selector=app=${TEST_K8S_RESSOURCE}
```

Comment:
Do we have the equivalent with the library? Can be useful for debug purpose.

Reply:
We don't have it, I will add that to the library.

Reply:
Done in https://github.com/nuxeo/platform-ci-shared-library/pull/6